### ISSUE TITLE
Goto definition not working

### ISSUE THREAD

**Issue by apj68** [marlon407/code-groovy#19]
I'm a relative newbie to vscode so maybe I'm missing it but F12 doesn't jump to function definition nor do I find other options.  Is it just me?

**Comment by bverkron** [comment]
I don't think this feature is implemented by this plugin, but it would be nice.

**Comment by mcouthon** [comment]
Do you know of any other plugin or built-in way in VSCode to have go-to-defintion in groovy?
